### PR TITLE
add @carlcauchi favorite restaurant

### PR DESCRIPTION
**Pull request by carlcauchi**

### CODE REVIEW

File: bios/carlcauchi.md [hunk 1]
```diff
@@ -2,3 +2,5 @@
 - my bio
 - bla bla bla
 - :heart:
+## Favourite Restaurant
+- dubai underwater restaurant
```
Comment (mrmorris) [inline bios/carlcauchi.md:6]: Yum

**Comment by mrmorris** [comment]
:+1:

**Comment by crichID** [comment]
@carlcauchi are you in Dubai? I was there a few years ago - amazing place!

File: bios/carlcauchi.md (hunk 1 above)
Comment (cheenpo) [inline bios/carlcauchi.md:6]: for reals?

**Comment by carlcauchi** [comment]
only i wish :) im in Malta at the moment

File: bios/carlcauchi.md [hunk 2]
```diff
@@ -2,3 +2,5 @@
 - my bio
 - bla bla bla
```
Comment (cheenpo) [inline bios/carlcauchi.md:3]: comment on blah blah blah line... to see what happens.

Comment (cheenpo) [inline bios/carlcauchi.md:3]: cool beans. that's pretty handy.  :+1:

**Comment by crichID** [comment]
Wow - late night for you - thank you for staying up to learn with us!

File: bios/carlcauchi.md (hunk 1 above)
Comment (carlcauchi) [inline bios/carlcauchi.md:6]: i never went, but im a fan of diving.. so that would be pretty cool :)

**Pull request merged**